cmd/istioctl/inject.go

```diff
@@ -105,6 +105,9 @@ kubectl get deployment -o yaml | istioctl kube-inject -f - | kubectl apply -f -
 			if versionStr == "" {
 				versionStr = version.Line()
 			}
+			if tag == "" {
+				tag = versionStr
```

Comment:
which one is what gets printed (for releases, short version) by "istioctl version" ?